test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -572,14 +572,15 @@ def _validate_pytorch_framework_version(request, processor, ecr_image, test_name
     }
     """
     if request.node.get_closest_marker(test_name):
-        _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
+        image_framework, image_framework_version = get_framework_and_version_from_tag(ecr_image)
         image_cuda_version = get_cuda_version_from_tag(ecr_image) if processor == "gpu" else ""
 
-        for framework_condition, processor_conditions in skip_dict.items():
-            if Version(image_framework_version) in SpecifierSet(framework_condition) and (
-                processor in processor_conditions or image_cuda_version in processor_conditions
-            ):
-                return True
+        if image_framework == "pytorch":
```

Comment:
may be redundant b/c this is in the pt training conftest, but thanks for making the change